test/test-utils/threads.ts

```diff
@@ -136,10 +136,6 @@ export const mkThread = ({
 
     const thread = room.createThread(rootEvent.getId()!, rootEvent, events, true);
 
-    events.forEach((event) => {
-        thread.timeline.push(event);
-    });
```

Comment:
this is a separate fix. It's redundant because `Room.createThread` already adds the events to the thread's timeline; adding them again results in lots of warnings from React about duplicate keys when we try to render the thread timeline.